core/interfaces.go

```diff
@@ -135,4 +135,5 @@ type StorageAuthority interface {
 // Publisher defines the public interface for the Boulder Publisher
 type Publisher interface {
 	SubmitToCT(ctx context.Context, der []byte) error
+	SubmitToSingleCT(ctx context.Context, logID string, der []byte) error
```

Comment:
That is a bummer, but I still think its worthwhile. In the SCT plan draft I've worked up (https://github.com/letsencrypt/boulder/wiki/Embedded-SCT-plan), the CA becomes the sole arbiter of which logs it cares about. I think experience has shown that two components having shared jurisdiction of config values that have to be the same is not great.

Perhaps we could solve the issue by having a cache of `LogClient`s and `SignatureVerifiers` based on `logURL`? Construct them the first time we need them? For stat name, I think we can pretty easily construct it live rather than caching it.